src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -47,14 +47,8 @@ Pane::Pane(const IPaneContent& content, const bool lastFocused) :
     // LOAD-BEARING: This will NOT work if the border's BorderBrush is set to
     // Colors::Transparent! The border won't get Tapped events, and they'll fall
     // through to something else.
-    _borderFirst.Tapped([this](auto&, auto& e) {
-        _FocusFirstChild();
-        e.Handled(true);
-    });
-    _borderSecond.Tapped([this](auto&, auto& e) {
-        _FocusFirstChild();
-        e.Handled(true);
-    });
+    _borderFirst.Tapped([this](auto& s, auto& e) { _borderTappedHandler(s, e); });
+    _borderSecond.Tapped([this](auto& s, auto& e) { _borderTappedHandler(s, e); });

```

Comment:
```suggestion
    _borderFirst.Tapped({ this, _borderTappedHandler });
    _borderSecond.Tapped({ this, _borderTappedHandler });
```
The construction without a lambda is more efficient, technically. But it's also a bit easier to read IMO.